src/constructs/iam/policies/parameter-store-read.ts

```diff
@@ -4,7 +4,11 @@ import type { GuStack } from "../../core";
 import type { AppIdentity } from "../../core/identity";
 import { GuPolicy } from "./base-policy";
 
-export class GuParameterStoreReadPolicyStatement extends PolicyStatement {
+/**
+ * This is helpful for reading all private configuration for a given app. For example, the
+ * [simple-configuration](https://github.com/guardian/simple-configuration) library requires these permissions.
+ */
+export class ReadParametersByPath extends PolicyStatement {
```

Comment:
I'm not sure that prefixing everything with `Gu` is particularly helpful (especially in the case of classes like this which we don't expect users to interact with directly), so I've dropped it here for brevity.

Reply:
Yeah, the `Gu` prefix is interesting. I can't find an ADR to explain our use of it. The closest I can find is https://github.com/guardian/cdk/blob/main/docs/architecture-decision-records/001-constructs-and-patterns.md:

> This library should define a number of Guardian flavoured constructs which extend those provided by the AWS CDK library with Guardian defaults baked in. For example, a `GuAutoScalingGroup` and `GuApplicationLoadBalancer`.

> specially in the case of classes like this which we don't expect users to interact with directly

Do we need to export them then?

Reply:
> Do we need to export them then?

Yes, unfortunately we need to use them slightly differently when configuring lambda permissions:

https://github.com/guardian/cdk/blob/3c9db96d2194f5a31e1533aa5f21bbbba345c2e9/src/constructs/lambda/lambda.ts#L122-L127